src/main/java/net/forgecraft/services/ember/util/serialization/ByteArrayResponseHandler.java

```diff
@@ -0,0 +1,21 @@
+package net.forgecraft.services.ember.util.serialization;
+
+import org.apache.hc.client5.http.impl.classic.AbstractHttpClientResponseHandler;
+import org.apache.hc.core5.http.HttpEntity;
+
+import java.io.IOException;
+
+public class ByteArrayResponseHandler extends AbstractHttpClientResponseHandler<byte[]> {
```

Comment:
This comes back to my original comment about the point of using this http library when the built in java one can do this just fine

Reply:
see above.